Roles/Neutral/Follower.cs

```diff
@@ -60,10 +60,10 @@ public override void Add(byte playerId)
         if (!Main.ResetCamPlayerList.Contains(playerId))
             Main.ResetCamPlayerList.Add(playerId);
     }
-    private static void SendRPC(byte playerId)
+    private void SendRPC(byte playerId)
     {
         MessageWriter writer = AmongUsClient.Instance.StartRpcImmediately(PlayerControl.LocalPlayer.NetId, (byte)CustomRPC.SyncRoleSkill, SendOption.Reliable, -1);
-        writer.WritePacked((int)CustomRoles.Follower); //SyncFollowerTargetAndTimes
+        writer.WritePacked(_state.PlayerId); //SyncFollowerTargetAndTimes
         writer.Write(playerId);
```

Comment:
Maybe instead of `_state.PlayerId` shold `_Player`

Reply:
lol that was from a regex find and edit. but I realized follower rpc isn't used. Ig I can activate it (idk if works properly tho since it wasn't previously used)